framadate/docker-compose.framadate.yml

```
@@ -0,0 +1,47 @@
+version: '2'
+
+networks:
+  framadate-internal:
+
+services:
+    framadate-db:
```

```suggestion
  framadate-db:
```

This should be indented by two spaces, not four.